Sources/Defaults/UserDefaults.swift

```diff
@@ -127,6 +54,12 @@ extension UserDefaults {
 			is Data.Type,
 			is Data?.Type:
 			return true
```

Comment:
Do we still need this now that we have the bridges? Couldn't we just rely on the bridges to do the correct thing?

For example, if we defined something like this:

```swift
extension Float: Defaults.NativelySupportedType {}
```

Reply:
I think the bridges should use to compile time check, and `isNativelySupportedType` is for runtime check?

Reply:
I'm not sure that is the case as most of this can be determined at compile time. Would be worth looking into at least.

For example, https://www.vadimbulavin.com/advanced-guide-to-userdefaults-in-swift/ is only using the protocol constraints.

Reply:
Yes. But in this project we still need `isNativelySupportedType` to specify  if the `Value` need to be serialized.
If we remove `isNativelySupportedType` only using the protocol constraints. we need to create bridge for all `Defaults.Serializable` like [SwiftyUserDefaults/DefaultsBridges.swift](https://github.com/sunshinejr/SwiftyUserDefaults/blob/master/Sources/DefaultsBridges.swift)

Reply:
Ah, right. Good point.